features/admin-file-attributes.feature

```diff
@@ -0,0 +1,142 @@
+    And I should see a file attribute table
+      | Attribute Name | Value     | Unit |
+      | last_modified  | 2013-7-15 |      |
+    And I should see a button input "Delete"
+    When I press "Delete"
+    And I press "Save"
+    Then I go to "dataset/100056"
+    And I should see "Termitomyces sp. J132 fungus genome assembly data."
+    And I follow "History"
+    And I should see "History" tab with text "Termitomyces_assembly_v1.0.fa.gz: file attribute deleted"
+
+  @ok @javascript @nonPublished
+  Scenario: Go to a non published dataset found in production-like database, should see a keyword attribute and a camera parameters attribute and their File Attribute Id
+    Given I sign in as an admin
+    And I am on "/adminFile/update/id/95354"
+    And I should see a file attribute table
+    | Attribute Name    | Value         | Unit |
+    | keyword           | test Bauhinia |      |
+    | camera parameters | test photo    |      |
+    When I go to "/adminFile/view/id/95354"
+    Then I should see field "File Attribute Id" with value "5441 5442 "
+
+  @ok @javascript @nonPublished
+  Scenario: Go to a non published dataset found in production-like database, delete a keyword attribute
+    Given I sign in as an admin
+    And I am on "/adminFile/update/id/95354"
+    When I press "Delete"
+    Then I should see a file attribute table
+    | Attribute Name    | Value         | Unit |
+    | camera parameters | test photo    |      |
+
+  @ok @javascript @nonPublished
+  Scenario: Go to a non published dataset found in production-like database, delete a keyword attribute and save
+    Given I sign in as an admin
+    And I am on "/adminFile/update/id/95354"
+    When I press "Delete"
+    And I press "Save"
+    Then I am on "/adminFile/view/id/95354"
+    And I should see field "File Attribute Id" with value "5442 "
+
+  @ok @javascript @nonPublished
+  Scenario: Go to a non published dataset found in production-like database, delete all attributes
+    Given I sign in as an admin
+    And I am on "/adminFile/update/id/95354"
+    When I press "Delete"
+    And I press "Delete"
+    Then I should not see "test Bauhinia"
+    And I should not see "test photo"
+
+  @ok @javascript @nonPublished
+  Scenario: Go to a non published dataset found in production-like database, delete all attributes and save
+    Given I sign in as an admin
+    And I am on "/adminFile/update/id/95354"
+    When I press "Delete"
+    And I press "Delete"
+    And I press "Save"
+    Then I am on "/adminFile/view/id/95354"
+    And I should see field "File Attribute Id" with empty value
+
+  @ok @javascript @nonPublished
+  Scenario: Go to a non published dataset found in production-like database,  delete a file attribute and check the last page of dataset log
+    Given I sign in as an admin
+    And I am on "/adminFile/update/id/95354"
+    When I press "Delete"
+    And I press "Save"
+    Then I go to "datasetLog/admin/DatasetLog_page/74"
```

Comment:
Deleting a file attribute in a non-published dataset create a new entry in the `curation_log` table. Is there a `CurationLog_page` that you could go to so that the Behat test can check that the `FCHCGJYBBXX-HKBAUpcgEAACRAAPEI-201_L2_2.fq.gz: file attribute deleted` action is displayed?

Reply:
There is no such page `CurationLog_page`. In fact, I was thinking, now we could check whether a file attribute has been deleted or not by checking the `File Attribute Id` on `admin file view` page, so checking the message in history tab could be removed?

Reply:
What you suggest is already tested in the above `Scenario: Delete all attributes and save, File Attribute Value on admin file view page should be empty`?

Reply:
Ok.